Userland/Libraries/LibSQL/Meta.cpp

```diff
@@ -10,78 +10,112 @@
 
 namespace SQL {
 
-ColumnDef::ColumnDef(String const& name, SQLType sql_type)
-    : Relation(name)
+/* ----------------------------------------------------------------------- */
```

Comment:
why do you use those lines?

Reply:
I use them to find my way around my source. I will delete them